crates/re_types/src/archetypes/points3d_ext.rs

```diff
@@ -0,0 +1,214 @@
+            Property::Short(v) => Some(v as u8),
+            Property::UShort(v) => Some(v as u8),
+            Property::Int(v) => Some(v as u8),
+            Property::UInt(v) => Some(v as u8),
+            Property::Float(v) => Some((v * 255.0) as u8),
+            Property::Double(v) => Some((v * 255.0) as u8),
+            Property::Char(v) => Some(v as u8),
+            Property::UChar(v) => Some(v),
+            Property::ListChar(_)
+            | Property::ListUChar(_)
+            | Property::ListShort(_)
+            | Property::ListUShort(_)
+            | Property::ListInt(_)
+            | Property::ListUInt(_)
+            | Property::ListFloat(_)
+            | Property::ListDouble(_) => None,
+        }
+    }
+
+    fn string(prop: &Property) -> Option<Cow<'_, str>> {
+        match prop {
+            Property::ListUChar(chars) => Some(String::from_utf8_lossy(chars)),
+            Property::ListChar(_)
+            | Property::ListShort(_)
+            | Property::ListUShort(_)
+            | Property::ListInt(_)
+            | Property::ListUInt(_)
+            | Property::ListFloat(_)
+            | Property::ListDouble(_)
+            | Property::Char(_)
+            | Property::UChar(_)
+            | Property::Short(_)
+            | Property::UShort(_)
+            | Property::Int(_)
+            | Property::UInt(_)
+            | Property::Float(_)
+            | Property::Double(_) => None,
+        }
+    }
+
+    struct Vertex {
+        position: Position3D,
+        color: Option<Color>,
+        radius: Option<Radius>,
+        label: Option<Text>,
+    }
+
+    // TODO(cmc): This could be optimized by using custom property accessors.
+    impl Vertex {
+        fn from_props(props: &LinkedHashMap<String, Property>) -> Option<Vertex> {
+            // NOTE: Empirical evidence points to these being de-facto standard…
+            const PROP_X: &str = "x";
+            const PROP_Y: &str = "y";
+            const PROP_Z: &str = "z";
+            const PROP_RED: &str = "red";
+            const PROP_GREEN: &str = "green";
+            const PROP_BLUE: &str = "blue";
+            const PROP_RADIUS: &str = "radius";
+            const PROP_LABEL: &str = "label";
+
+            let (Some(x), Some(y), Some(z)) = (
+                props.get(PROP_X).and_then(f32),
+                props.get(PROP_Y).and_then(f32),
+                props.get(PROP_Z).and_then(f32),
+            ) else {
+                return None;
+            };
+
+            let mut this = Self {
+                position: Position3D::new(x, y, z),
+                color: None,
+                radius: None,
+                label: None,
+            };
+
+            if let (Some(r), Some(g), Some(b)) = (
+                props.get(PROP_RED).and_then(u8),
+                props.get(PROP_GREEN).and_then(u8),
+                props.get(PROP_BLUE).and_then(u8),
+            ) {
+                this.color = Some(Color::new((r, g, b)));
+            };
+
```

Comment:
I suggest adding support for "alpha" as well (if it exists), even though the renderer currently ignores it